server/tests/unit/services/RunService.test.js

```diff
@@ -224,18 +224,421 @@ describe('RunService', () => {
+                    at_version_id: atVersion.id,
+                    browser_version_id: browserVersion.id,
+                    active: true,
+                    run_status_id: runStatus.id
+                });
+
+                // Create an active run
+                const previouslyActiveRunEntry = await db.Run.create({
+                    browser_version_id: browserVersion.id,
+                    at_version_id: atVersion.id,
+                    at_id: at.id,
+                    test_cycle_id: testCycle.id,
+                    browser_version_to_at_versions_id: tech.id,
+                    apg_example_id: apgExamples[0].id,
+                    test_version_id: testVersion.id,
+                    active: true,
+                    run_status_id: runStatus.id
+                });
+
+                // Create an inactive run
+                const previouslyInactiveRunEntry = await db.Run.create({
+                    browser_version_id: browserVersion.id,
+                    at_version_id: atVersion.id,
+                    at_id: at.id,
+                    test_cycle_id: testCycle.id,
+                    browser_version_to_at_versions_id: tech.id,
+                    apg_example_id: apgExamples[1].id,
+                    test_version_id: testVersion.id,
+                    active: false,
+                    run_status_id: runStatus.id
+                });
+
+                let previouslyActiveRun = await db.Run.findOne({
+                    where: { id: previouslyActiveRunEntry.id }
+                });
+                let previouslyInactiveRun = await db.Run.findOne({
+                    where: { id: previouslyInactiveRunEntry.id }
+                });
+
+                const activeRuns = await RunService.configureRuns({
+                    test_version_id: testVersion.id,
+                    apg_example_ids: apgExamples
+                        .map(example => example.id)
+                        .slice(1), // remove the first element of examples
+                    at_browser_pairs: [
+                        {
+                            at_name_id: at.at_name_id,
+                            at_version: atVersionNumber,
+                            browser_id: browser.id,
+                            browser_version: browserVersionNumber
+                        }
+                    ],
+                    run_status_id: runStatus.id
+                });
+
+                await previouslyActiveRun.reload();
+                expect(previouslyActiveRun.active).toBe(false);
+
+                await previouslyInactiveRun.reload();
+                expect(previouslyInactiveRun.active).toBe(true);
+
+                const newlyCreatedRun = Object.values(activeRuns).filter(
+                    run =>
+                        run.id !== previouslyActiveRun.id &&
+                        run.id !== previouslyInactiveRun.id
+                );
+                expect(newlyCreatedRun.length).toBe(1);
```

Comment:
Can you actually test that they the correct values.

Reply:
https://github.com/w3c/aria-at-app/pull/174/commits/2c11bfd661be27d3010bcaf51c94e68e6aa3ad3a#diff-3e074bfb22ba34ca6d7b9412c4ae20eeR626